### PR TITLE
Adds missing RenderBufferFlush Call

### PR DESCRIPTION
**Pull request by Kuranes**

### CODE REVIEW

File: sources/osgViewer/View.js [hunk 1]
```diff
@@ -256,6 +256,7 @@ View.prototype = {
         availableTime = Program.flushDeletedGLPrograms( gl, availableTime );
         availableTime = Shader.flushDeletedGLShaders( gl, availableTime );
         availableTime = FrameBufferObject.flushDeletedGLFrameBuffers( gl, availableTime );
+        availableTime = FrameBufferObject.flushDeletedGLRenderBuffers( gl, availableTime );
```
Comment (jtorresfabra) [inline sources/osgViewer/View.js]: You mean RenderBufferObject no? ;)

Comment (Kuranes) [inline sources/osgViewer/View.js]: yes :) needs fixing

**Comment by Kuranes** [comment]
Finally used FrameBufferObject as it's not really clear as the current code is which is responsible for what.
RenderBufferObject wasn't really a class on its own, needing its own js file, as it's handled really by the FrameBufferObjectClass.

(fbo is under rework in the PR about renderbin RTT, and will soon be under more work again, multidraw buffer ftw !)

**Comment by jtorresfabra** [comment]
Great to hear that! :+1:

**Comment by Kuranes** [comment]
# _**Webgl2 is coming**_

**Pull request merged**